### PR TITLE
clean up Spyglass metadata viewer

### LINKED ISSUE #9494: amusing spyglass metadata bug
screen shot 2018-09-19 at 6 44 51 pm
/kind bug
/shrug
/this-is-fine

/assign @ibzib @paulangton

### LINKED ISSUE #9499: TODO in Spyglass metadata viewer
The Tests field for the Spyglass metadata viewer contains a hard-coded TODO stamp. @paulangton what was intended to go here?

screenshot from 2018-09-20 10-59-26

### PR DESCRIPTION
**Pull request by ibzib**
/cc @paulangton @BenTheElder 
fixes #9494 
fixes #9499

Before:
screenshot from 2018-09-20 11-01-45

After:
screenshot from 2018-09-20 15-17-53

### CODE REVIEW

File: prow/spyglass/viewers/metadata/metadata_viewer.go [hunk 1]
```diff
@@ -98,10 +98,12 @@ func ViewHandler(artifacts []viewers.Artifact, raw string) string {
 		}
 
 	}
-	d := Derived{
-		Elapsed: metadataViewData.Finished.Timestamp.Sub(metadataViewData.Started.Timestamp),
+	if metadataViewData.Finished.TimestampRaw == 0 {
+		metadataViewData.Derived.Elapsed = time.Now().Sub(metadataViewData.Started.Timestamp)
+	} else {
+		metadataViewData.Derived.Elapsed =
```
Comment (BenTheElder) [inline prow/spyglass/viewers/metadata/metadata_viewer.go]: perhaps instead we should render from a string, and put something like "(running)" at the end when we don't have a finished time?

Comment (BenTheElder) [reply prow/spyglass/viewers/metadata/metadata_viewer.go]: (or the template can handle this)

**Comment by stevekuznetsov** [comment]
While we're touching this, can we get a `X minutes ago` with a mouseover for the time in the user's timezone?

**Comment by ibzib** [comment]
@stevekuznetsov I'll make a note to put that in after #8967 is resolved.

**Comment by krzyzacy** [comment]
(is this one good to go?)

**Comment by ibzib** [comment]
This one should be good to go as well

**Pull request merged**